src/components/PlaceList/PlaceCard.tsx

```diff
@@ -36,13 +59,19 @@ const PlaceCard: React.FC<PlaceCardProps> = ({ place, avg, address, photoref }:
 			accessibilityLabel={place ? `No image available for ${place}` : "No image available"}
 		/>
 	);
+
 	return (
 		<View style={styles.card}>
 			<View style={styles.alignText}>
 				<Text ellipsizeMode="tail" numberOfLines={1} style={styles.title}>
 					{place}
 				</Text>
-
+				<Text
+					accessibilityLabel={distanceInMi ? `${distanceInMi} miles away` : ""}
```

Comment:
do you need an accessibility label if it's the exact same as the text? will this make it read out twice?

Reply:
> the accessibility label will only be read once regardless of the actual text (adding a label overrides it)

> also its slightly different, the text uses "mi" while the label reads out "miles

from Discord server 2021-11-05